modules/images/webp-uploads/load.php

```diff
@@ -683,3 +705,79 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 	return rest_ensure_response( $data );
 }
 add_filter( 'rest_prepare_attachment', 'webp_uploads_update_rest_attachment', 10, 3 );
+
+/**
+ * Returns the attachment sources array ordered by filesize.
+ *
+ * @since n.e.xt
+ * @private
```

Comment:
No need to use the `@private` tag since this function is public. Private functions in WordPress start with `_`, so if you want to make it private then it should be `_webp_uploads_get_attachment_sources` (not sure if it is really needed though).